test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua

```diff
@@ -0,0 +1,148 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [GeneralResultCodes] GENERIC_ERROR watchdog timeout from HMI
+-- [HMI API] UI.PerformAudioPassThru request/response
+-- [HMI API] TTS.Speak request/response
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case SDL splits the request from mobile app to one and/or more HMI interfaces
+-- and at least one of the interfaces does not respond during SDL`s watchdog
+-- (important note: this component is working and has responded to previous RPCs)
+-- SDL must: return "GENERIC_ERROR, success: false" result to mobile app and
+-- include appropriate description into "info" parameter.
+-- In this case info = "'%component-name%' component does not respond"
+--
+-- 1. Used preconditions
+-- 1.1. PerformAudioPassThru RPC is allowed by policy
+-- 1.2. All params used in PerformAudioPassThru_request are present and within bounds
+-- 1.3. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, other params)
+-- HMI sends to SDL UI.PerformAudioPassThru (SUCCESS)
+-- HMI does not send to SDL TTS.Speak
+--
+-- Expected result:
+-- SDL sends UI.PerformAudioPassThru (audioPassThruIcon, other params) to HMI
+-- SDL sends to HMI UI.PerformAudioPassThru
+-- SDL sends to mobile PerformAudioPassThru (GENERIC_ERROR, success:false, info = "TTS.Speak component does not respond")
```

Comment:
Clarified in `Clarifications about result codes for PerformAudioPassThru` and updated